src/styles/exports/overrides.css

```diff
@@ -580,12 +582,12 @@ sl-dialog.small::part(body) {
     --header-spacing: 32px 0 16px 24px;
   }
 
-  sl-dialog.has-icon > div {
+  sl-dialog.has-icon>div {
```

Comment:
The syntax highlighting implies the lack of spaces around `>` is fine, just calling it out to be sure :)

Reply:
I had initially added these files to .prettierignore, but trying out putting them back in

Reply:
I ended up being able to just ignore one of the CSS files (the generated one), which for whatever reason fails the prettier check on CI, but not locally.